statsmodels/stats/tests/test_power.py

```diff
@@ -727,28 +727,6 @@ def test_power_solver_warn():
     nip.start_bqexp['nobs1'] = {'upp': 50, 'low': -20}
     val = nip.solve_power(0.1, nobs1=None, alpha=0.01, power=pow_, ratio=1,
                           alternative='larger')
-    import scipy
-    if scipy.__version__ < '0.10':
```

Comment:
Passes tests on 0.12.  0.10 isn't supported.


On Mon, Jun 20, 2016 at 11:39 PM Josef Perktold <notifications@github.com>
wrote:

> In statsmodels/stats/tests/test_power.py
> <https://github.com/statsmodels/statsmodels/pull/3049#discussion_r67779829>
> :
>
> > @@ -727,28 +727,6 @@ def test_power_solver_warn():
> >      nip.start_bqexp['nobs1'] = {'upp': 50, 'low': -20}
> >      val = nip.solve_power(0.1, nobs1=None, alpha=0.01, power=pow_, ratio=1,
> >                            alternative='larger')
> > -    import scipy
> > -    if scipy.__version__ < '0.10':
>
> AFAIR, this shouldn't be deleted, the problem was with 0.12 which failed
> compared to older versions.
> This needs checking if it works with recent scipy.
> i.e. need to check changes in scipy optimize.
>
> IIRC
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/statsmodels/statsmodels/pull/3049/files/0b3b497521efdb7bac4cb1ef9a121b6604e86081#r67779829>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFU5RcKbTHE7tdMuKPdkYRx3MRiVWyziks5qNxargaJpZM4I6DYw>
> .
>